modules/carto/src/layers/vector-tile-layer.ts

```diff
@@ -148,22 +145,9 @@ export default class VectorTileLayer<
     return subLayer;
   }
 
-  getPickingInfo(params) {
-    const info = super.getPickingInfo(params);
-
-    // MVT tiles use tile-relative coordinates, handled by MVTLayer#getPickingInfo.
-    if (this.state.mvt) {
-      return info;
-    }
-
-    // CARTO tiles use [lat, lng], so overwrite `info.object`.
-    if (this.state.binary && info.index !== -1) {
-      const {data} = params.sourceLayer!.props;
-      info.object = binaryToGeojson(data as BinaryFeatureCollection, {
-        globalFeatureId: info.index
-      }) as Feature<Geometry, FeaturePropertiesT>;
-    }
-
-    return info;
+  protected override _isWGS84(): boolean {
+    // MVT coordinates are tile-relative; CARTO binary coordinates are [lat, lng].
```

Comment:
I think they are XY ordered. Can you double-check?
```suggestion
    // MVT coordinates are tile-relative; CARTO binary coordinates are [lng, lat].
```

Reply:
Confirmed lng,lat in the binary data with VectorTileLayer, thanks!